contracts/managers/MACOStrategyManagerV2.sol

```diff
@@ -0,0 +1,611 @@
+/*
+    Copyright 2019 Set Labs Inc.
```

Comment:
Can you highlight the diff between this and V1?

Reply:
I'm assuming its just the `riskAssetOracleInstance.read` and using the `IOracle` interface

Reply:
And passing in riskAssetOracleInstance instead of querying the address through the `HistoricalPriceFeed`. To make that work and avoid stack depth issues we have to pass the collateral tokens in, in an array.